Formula/extra-cmake-modules.rb

```diff
@@ -1,4 +1,4 @@
-class KdeExtraCmakeModules < Formula
+class ExtraCmakeModules < Formula
```

Comment:
This one seems a little ambiguous. What do other distros call this package? Can we call it `KextraCmakeModules` maybe?

Reply:
I don't think they're ambiguous, they're not KDE-specific and contain many useful CMake modules. From the project's own description: _"While the driving force of this module is to reduce duplication in CMake scripts across KDE software, it is intended to be useful for any software that uses the CMake build system."_

Also, the vast majority calls it exactly as the upstream does: https://repology.org/project/extra-cmake-modules/versions

Reply:
It's a little ambiguous in that the name makes it sound like the formula is for software from KitWare, but it isn't really. However, if that's what other distros call it, then that's fine.

Reply:
For what it's worth, the alternative, common description of the project is _"Extra CMake Modules, or ECM, aims to augment CMake with additional modules. It serves as both a staging ground for new modules before they are moved upstream to CMake and a place for modules that, for whatever reason, are not a good fit for CMake itself."_

Many of the ECM modules/improvements were eventually migrated to the CMake itself, and it is a de-facto extension of the CMake itself.